package.json

```diff
@@ -49,6 +49,7 @@
     "ngx-loading": "^1.0.14",
     "ngx-quill": "^3.3.0",
     "popper.js": "^1.14.3",
+    "react-progressive-image-loading": "^3.0.3",
```

Comment:
Is this intended to be here? Shouldn't be mixing frameworks, so better not to use React libraries.